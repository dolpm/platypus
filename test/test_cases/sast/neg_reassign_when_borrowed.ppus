pipe main |> [] |> [] |> unit {
  mut int y <| 1;
  &int x <| &y;
  y <| 10;
  |> ();
}