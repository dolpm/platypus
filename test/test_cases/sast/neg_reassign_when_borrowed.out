Fatal error: exception Failure("variable y can't be reassigned as it is already borrowed.")
